### PR TITLE
tofi: init at 0.5.0

### PR DESCRIPTION
**Pull request by fbergroth**
###### Description of changes

Packaged [tofi](https://github.com/philj56/tofi), a tiny dynamic menu for Wayland.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/tofi/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, ninja, meson, scdoc
+, wayland-protocols, freetype, harfbuzz, cairo, pango, wayland, libxkbcommon }:
+stdenv.mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/tofi/default.nix:3]:
```suggestion
, wayland-protocols, freetype, harfbuzz, cairo, pango, wayland, libxkbcommon
}:

stdenv.mkDerivation rec {
```

**Review by SuperSandro2000 — APPROVED** [review]

File: pkgs/applications/misc/tofi/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  nativeBuildInputs = [ meson ninja pkg-config scdoc wayland-protocols ];
+  buildInputs = [ freetype harfbuzz cairo pango wayland libxkbcommon ];
+
+  meta = with lib; {
+    description = "Tiny dynamic menu for Wayland";
+    homepage = "https://github.com/philj56/tofi";
+    license = licenses.mit;
+    maintainers = with maintainers; [ fbergroth ];
+    platforms = platforms.linux;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/tofi/default.nix:22]:
```suggestion
    platforms = platforms.linux;
    broken = stdenv.isAarch64;
```

see ofborg

**Comment by fbergroth** [comment]
Thanks for the review, I ran it through nixpkgs-fmt and added meta.broken as you suggested.

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Pull request merged**